core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -173,6 +175,31 @@ public static void merge(Map<?, ?> properties) {
     checkUserFileBufferBytes();
   }
 
+  /**
+   * Iterates a set of site properties and discards those that user should not use.
+   *
+   * @param siteProperties the set of site properties to check
+     */
+  public static void discardIgnoredSiteProperties(Map<?, ?> siteProperties) {
+    if (siteProperties != null) {
+      Iterator<? extends Map.Entry<?, ?>> iter = siteProperties.entrySet().iterator();
+      while (iter.hasNext()) {
+        Map.Entry<?, ?> entry  = iter.next();
+        String key = entry.getKey().toString();
+        if (PropertyKey.isValid(key)) {
+          PropertyKey propertyKey = PropertyKey.fromString(key);
+          if (propertyKey.isIgnoredSiteProperty()) {
+            iter.remove();
+            LOG.warn("Site property '{}' is deprecated. Its value '{}' will be ignored. "
+                + "Use corresponding system properties, JVM properties and environment variables "
+                + "instead. Otherwise, Alluxio will use default value '{}'",
```

Comment:
this is not accurate - we will prefer system properties over the default value

Reply:
Done.